docs/tutorials/light-up.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Use Object Detection to Turn Your Lights On"
+linkTitle: "Turn on Lights with Object Detection"
+weight: 49
+type: "docs"
+tags: ["camera", "vision", "detector", "python"]
+description: "How to turn a light on when your webcam sees a person."
+# Author: Hazal Mestci
+---
+
+This tutorial uses the Viam [Vision Service](/services/vision/) with your computer's built-in webcam to detect the presence of a person and turn a lamp on when you sit down at your desk.
+
+You can turn it into a night light for reading books, a security robot that alerts you when a person is close by, or a bathroom light that only activates when people enter; the opportunities are endless.
+
+This project is a great place to start if you are new to building robots, and have a smart plug laying around.
+
+{{< gif webm_src="../img/light-up/light-up.webm" mp4_src="../img/light-up/light-up.mp4" alt= "The project working: a person sitting at a desk with a computer and lightbulb set up in front of her. As she leaves the light turns off, and as she enters the frame, the light turns back on." >}}
+
+## Hardware requirements
+
+You need the following hardware for this tutorial:
+
+- Computer with a webcam
+  - This tutorial uses a MacBook Pro but any computer running macOS or 64-bit Linux will work
+- Mobile phone (to download the Kasa Smart app)
+- [Kasa Smart Wi-Fi Plug Mini](https://www.kasasmart.com/us/products/smart-plugs/kasa-smart-wifi-plug-mini)
+- [Kasa Smart Light Bulb](https://www.kasasmart.com/us/products/smart-lighting/kasa-smart-light-bulb-multicolor-kl125)
+- [Table Lamp Base](https://www.amazon.com/gp/product/B08KZNZVY7/ref=ppx_yo_dt_b_asin_title_o02_s00?) or similar
+
+## Software requirements
+
+The following software is used in this tutorial:
+
+- [Python 3.8 or newer](https://www.python.org/downloads/)
+- [`viam-server`](/installation/install/)
+- [Viam Python SDK](https://python.viam.dev/)
+  - The Viam Python SDK (software development kit) lets you control your Viam-powered robot by writing custom scripts in the Python programming language.
+  Install the Viam Python SDK by following [these instructions](https://python.viam.dev/).
+- [Project repo on GitHub](https://github.com/viam-labs/devrel-demos/tree/main/Light%20up%20bot)
+
+## Install `viam-server` and connect to your robot

```

Comment:
if we are leaving all other python instructions out, why are we keeping the viam-server one? because now it looks like software requirements are prerequisites, and viam-server step is there too

Reply:
I see your point, and this is a good question across all the tutorials. I feel like this is okay here because viam-server is more Viam-specific? And I mostly left it in because the way the setup tab works, viam-server ends up being intimately connected with creating a new robot, which I'm assuming people need to do for each tutorial, whereas maybe they only need to install Python once.